docs/source/installation.rst

```diff
@@ -2,147 +2,123 @@ Installation
 ============
 
 Supported OS
-------------
+++++++++++++
 
-Currently, we only support ``MacOS`` and ``Linux`` operating systems. ``Windows``
-users have the possibility to install and use ``ivadomed`` via
-`Windows Subsystem for Linux (WSL) <https://docs.microsoft.com/en-us/windows/wsl/>`_. The steps below (about updating bashrc) are strongly recommended for MacOS users in the installation process but are optional for Linux users.
+Currently, ``ivadomed`` only supports GPU/CPU on ``Linux`` and ``Windows`` and CPU only on ``macOS`` and `Windows Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/>`_.
 
-Open your bash/zsh script file with editor on your computer.
+Step 1: Setup dedicated python environment
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-    If you are using bash shell
-    ::
+You can setup ``ivadomed`` using either Conda or Venv:
 
-        vim ~/.bashrc
+.. tabs::
 
-    If you are using zsh shell
-    ::
-        
-        vim ~/.zshrc
+    .. tab:: Install via ``conda``
 
-Write in your .bashrc/.zshrc file with following line.
+        This is the easiest way for personal computers.
 
-::
+        1. Create new conda environment using ``environment.yml`` file
 
-    export HDF5_USE_FILE_LOCKING='FALSE'
+        ::
 
-Save this change and restart your terminal to apply the change.
+            conda env create --file environment.yml
```

Comment:
Now that I think about it, I'm a bit confused about the purpose of this `environment.yml` file. 

https://github.com/ivadomed/ivadomed/blob/c226c49c384a8c322b99f9529f10b408b6687637/environment.yml#L1-L9

Since all of the dependencies are installed via `pip` + `requirements.txt`, it seems redundant with Step 3 of this page, where `requirements.txt` will be installed once more.

(Also, I notice the name is still hardcoded as `IvadoMedEnv` in this file, so it would probably need to be updated to `ivadomed_env` as well.)

Reply:
Good point. I think the original rationale was to actually have a one stop installation for all dependencies including torch etc (as those distribute via conda rather well) but then the issue with that is, GPU are not always supported and also we won't know at installation time which CUDA to use etc, which significantly complicate the matter. 

I like your suggestion to keep it simple so I will just keep the steps simple here to create an empty environment and leave the rest alone (removing pip installation part)